packages/bpk-component-navigation-bar/src/BpkNavigationBarIconButton.tsx

```diff
@@ -19,7 +19,7 @@
 import type { ComponentType, MouseEvent, ReactNode } from 'react';
 
 // @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
-import BpkIconButton from '../../bpk-component-close-button';
+import BpkCloseButton from '../../bpk-component-close-button';
```

Comment:
Wrong name❓

Reply:
Is the `expect-error` still valid with this changed?

Reply:
Yeah the close-button has not been ported to typescript yet. So still needed.